src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/commands.py

```diff
@@ -10,6 +10,9 @@
 
 from ._client_factory import web_client_factory
 
+def output_slots_In_table(slots):
+    return [{'name': s['name'], 'status': s['state'], 'app service plan': s['appServicePlan']} for s in slots]
```

Comment:
Yep, let us go with`plan`, i like shorter name and we are already under the context of `app service`. I used `app service plan` simply because portal does it, but this is not a very necessary alignment we have to do